src/main/java/org/jabref/gui/fieldeditors/LinkedFileViewModel.java

```diff
@@ -326,8 +326,13 @@ public boolean delete(FileDirectoryPreferences prefs) {
     }
 
     public void edit() {
-        FileListEntryEditor editor = new FileListEntryEditor(linkedFile, false, true, databaseContext);
-        SwingUtilities.invokeLater(() -> editor.setVisible(true, false));
+        LinkedFilesWrapper wrapper = new LinkedFilesWrapper();
+        wrapper.setLinkedFile(this.linkedFile);
+        LinkedFileEditDialogView dialog = new LinkedFileEditDialogView(wrapper);
+        if (dialog.showAndWait()) {
+            // TODO: What do when saved?
```

Comment:
@tobiasdiez  Do you have an idea how I can store the changes to the linkedFile I made in the dialog?
I can get back a new linkedFile object but no idea how to apply the changes to the orginal obj

Reply:
It should suffice to use the setters of the old linked file (the changes should then automatically be stored in the entry; at least if the linked file is already shown in the entry editor).